Makefile

```diff
@@ -18,6 +18,7 @@ CHARTS_RELEASE        := wire-server databases-ephemeral fake-aws aws-ingress ba
 BUILDAH_PUSH          ?= 0
 KIND_CLUSTER_NAME     := wire-server
 BUILDAH_KIND_LOAD     ?= 1
+export HSPEC_OPTIONS = --fail-on-focused
```

Comment:
A comment would be nice, on what this does and how this interacts with unit or integration tests, and why this gets set globally in the top-level makefile but not inside e.g. the services/brig/Makefile. (what are the implications of that for people using one or the other makefile to run tests?)

Also, this focussed override in the makefile and the nix dependencies looks like it could live in a separate PR to the wire-api-federator mock changes.

Reply:
> A comment would be nice, on what this does and how this interacts with unit or integration tests, and why this gets set globally in the top-level makefile but not inside e.g. the services/brig/Makefile. (what are the implications of that for people using one or the other makefile to run tests?)

I will add a comment.

> Also, this focussed override in the makefile and the nix dependencies looks like it could live in a separate PR to the wire-api-federator mock changes.

Hmm, I was in the mindset of fixing all the problems addition of this test had created. Creating many small PRs is a little annoying, but I guess that is our process, so in hind sight I agree with you but maybe we should figure out a less annoying process.